src/components/App.js

```diff
@@ -9,13 +9,33 @@ class App extends Component {
   constructor() {
       super()
       this.state = {
-        movies: movieData.movies,
-        currentMovie: {}
+        movies: {},
+        currentMovie: {},
+        error: ''
       }
   }
 
+  componentDidMount = () => {
+    // fetch('https://httpstat.us/500')
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => {
+      if(!response.ok) {
+        console.log('HTTP request unsuccessful');
+        this.setState({error: `Network Error - status ${response.status} at URL: ${response.url}`});
+        throw new Error(`status ${response.status} at URL: ${response.url}`)
+      } else {
+        console.log('HTTP request successful');
+      }
+      
+      return response;
+    })
+    .then(response => response.json())
+    .then(movies => this.setState({ movies: movies }))
+    .catch(err => console.log(err));
+  }
+
   loadMovieDetails = (id) => {
-    const selectedMovie = this.state.movies.find( movie => {
+    const selectedMovie = this.state.movies.movies.find( movie => {
```

Comment:
`movies.movies` is a little confusing. Seems like you actual want `this.state.movies.find` but there is something you're working through? You should be able to have `this.state.movies.find` if that's what you want.